semgrep/semgrep/notifications.py

```diff
@@ -0,0 +1,30 @@
+from semgrep.util import with_color
+from semgrep.verbose_logging import getLogger
+
+HAS_SHOWN_SETTINGS_KEY = "has_shown_metrics_notification"
```

Comment:
Should we have the settings keys be defined in settings.py or have it defined on the files that use settings?

Reply:
@nbrahms

Reply:
I have no opinions here

Reply:
Same. Let's keep it here then :D